color/src/colorspace.rs

```diff
@@ -355,6 +355,69 @@ impl ColorSpace for A98Rgb {
     }
 }
 
+/// 🌌 The Prophoto RGB color space.
+///
+///
+/// Prophoto RGB is similar to [sRGB](`Srgb`) but has higher red, green and blue chromaticities,
+/// thereby extending its gamut over sRGB on all components.
```

Comment:
Mention it has a different white point? We don't mention the white point of other color spaces, but as (linear) sRGB is central, maybe we should mention it here.

Reply:
Yes, since it has a white point that differs from sRGB, it's worth stating, but we don't need to make a big deal about it.